### DISCUSSION TITLE
番茄小说有些小说不是文字，请处理一下

### DISCUSSION THREAD

**Discussion by 13316282905** [xiaohucode/yidaRule#86] [Ideas]
![Screenshot_2024-03-01-19-41-54-129_com xinqiu yidaapp](https://github.com/xiaohucode/yidaRule/assets/159050612/08812692-d081-4dd3-8a04-c1642afb3eaa)

**Comment by xiaohucode** [comment]
最新规则已修复，重新导入

**Reply by 13316282905** [reply]
如何导入，是卸载再换新的吗

**Reply by 13316282905** [reply]
请问重新导入是指删了旧的安装新的输入软件显示的规则吗

**Reply by 13316282905** [reply]
作者在吗
请问重新导入是指删除旧的安装新的时候输入软件中新的规则吗
